README.md

```diff
@@ -21,9 +21,9 @@ bower install chart.js --save
 
 #### Selecting the Correct Build
 
-Chart.js provides two different builds that are available for your use. The `Chart.js` and `Chart.min.js` files include Chart.js and the accompanying color parsing library. If this version is used and you require the use of the time axis, [Moment.js](http://momentjs.com/) will need to be included before Chart.js.
+Chart.js provides two builds: `Chart.js`, `Chart.min.js`. Both `Chart.js` and `Chart.min.js` files include Chart.js as well as the color parsing library. If this version is used, you require to include [Moment.js](http://momentjs.com/) before Chart.js for the functionality of the time axis.
 
-The `Chart.bundle.js` and `Chart.bundle.min.js` builds include Moment.js in a single file. This version should be used if you require time axes and want a single file to include, select this version. Do not use this build if your application already includes Moment.js. If you do, Moment.js will be included twice, increasing the page load time and potentially introducing version issues.
+Both `Chart.bundle.js` and `Chart.bundle.min.js` builds include Moment.js in a single file. You should use this version if you require time axes and want to include a single file. You should not use this build if your application already included Moment.js. Otherwise, Moment.js will be included twice which results increasing page load time and possible version compatability issues.
```

Comment:
"which results increasing page load" -> "which results in increasing page load"

Reply:
Agreed